tests/Spec.hs

```diff
@@ -1003,7 +1003,7 @@ main = hspec $ do
         [ AIsReviewer "deckard"
         , ALeaveComment prId "<!-- Hoff: ignore -->\nPull request approved for merge and tag by @deckard, rebasing now."
         , ATryIntegrate "Merge #1: Untitled\n\nApproved-by: deckard\nAuto-deploy: false\n"
-                        (PullRequestId 1, Branch "refs/pull/1/head", Sha "abc1234") [] False
+                        (PullRequestId 1, Branch "refs/pull/1/head", Sha "abc1234") [] True
```

Comment:
Oh magical boolean, what do you mean. If it's not too much trouble would you mind changing this out for a 2 constructor type with better named constructors

Reply:
The magic boolean [already has a name](https://github.com/channable/hoff/blob/76137f291634204d2003457fcf25c2ad756df334/tests/Spec.hs#L109) but we use shorthand record construction here so the name doesn't show. I think it's reasonable to have a yes/no decision be a boolean but have no strong feelings about this.

Reply:
Ah nice, if it's possible to find out what magic boolean means then you can leave this for now. Personally I would still change it since you can easily use it in places like this where the record field doesn't appear to help you out